lua/starfall/libs_sh/holograms.lua

```diff
@@ -84,6 +117,7 @@ instance:AddHook("initialize", function()
 end)
 
 local function hologramOnDestroy(holo)
+	if CLIENT then clearParentFix(holo) end
```

Comment:
Maybe the `if CLIENT` can be moved inside these functions? Otherwise, I think setParent needs these `if CLIENT` checks too

Reply:
These functions are never used serverside though, `Hologram.setParent` is only defined on the client, since to parent holos on the server you just use `Entity.setParent`.  
Have I misunderstood what you meant or can this conversation be resolved?

Reply:
All right, that makes sense.